at_client/src/main/java/org/atsign/client/cli/Onboard.java

```diff
@@ -69,10 +78,35 @@ public static void main(String[] args) throws Exception {
         System.out.println("Storing encryption public key");
         onboarding.storePublicEncryptionKey(conn, atSign, keys);
 
-        // and as we've successfully authenticated with PKAM, let's delete the CRAM secret
+        // and as we've successfully authenticated with PKAM, let's delete the CRAM
+        // secret
         System.out.println("Deleting CRAM secret");
         onboarding.deleteCramKey(conn);
 
         System.out.println("Onboarding complete");
     }
+
+    public static String retrySecondaryConnection(String rootUrl, AtSign atSign)
```

Comment:
@gkc this is a change I've made to accommodate the fact that the secondary is not spun up immediately after cramkey is generated. Is this acceptable or is there a better way to do this